composer.json

```diff
@@ -22,13 +22,13 @@
     "require": {
         "php": "^7.4",
         "friendsofsymfony/oauth-server-bundle": ">2.0.0-alpha.0 ^2.0@dev",
-        "sensiolabs/security-checker": " ^5.0",
```

Comment:
Now its deprecated: https://github.com/sensiolabs/security-checker but is possible to use Symfony CLI tool instead of this lib